guides/release/upgrading/editions.md

```diff
@@ -363,14 +438,18 @@ export default class ApplicationController extends Controller {
 }
 ```
 
+The `init` hook still exists on many existing classes, and runs _after_
+`constructor`, so you can generally convert to native class syntax without
+rewriting your `init` methods. However, in the future `init` will be removed,
+so you should eventually transition to `constructor`.
+
```

Comment:
Should `@classic` be mentioned?

Reply:
Possibly, I think it'd be best to add in a followup though